src/main/java/lotto/domain/LottoPlay.java

```diff
@@ -41,8 +45,17 @@ public LottoNumbers createLottoNumber() {
             lottoNumber.add(new LottoNumber(LOTTO_NUMBER.get(i)
                                                         .getLottoNumber()));
         }
-        Collections.sort(lottoNumber);
 
         return new LottoNumbers(lottoNumber);
     }
+
+    public void createLottoByManual(List<Integer> manualLottoNumber) {
+        List<LottoNumber> lottoNumber = new ArrayList<>();
+
+        for (Integer number : manualLottoNumber) {
+            lottoNumber.add(new LottoNumber(number));
+        }
+
+        totalLottoNumbers.recordEachLottoNumbers(new IssueNumber(issueNumber++), new LottoNumbers(lottoNumber));
```

Comment:
```suggestion
        List<LottoNumber> lottoNumber = manualLottoNumber.stream().map(LottoNumber::new).collect(Collectors.toList());
        totalLottoNumbers.recordEachLottoNumbers(new IssueNumber(issueNumber++), new LottoNumbers(lottoNumber));
    }
```

이렇게도 표현이 가능한 것 같아요 : ) 참고만 해주세용!

Reply:
네 반영했습니다!